src/libs/actions/Report.js

```diff
@@ -96,6 +96,21 @@ function getParticipantEmailsFromReport({sharedReportList}) {
     return _.without(emailArray, currentUserEmail);
 }
 
+/**
+ * Returns a generated report title based on the participants
+ *
+ * @param {Array} sharedReportList
+ * @return {String}
+ */
+function getChatReportName(sharedReportList) {
```

Comment:
I'm confused why this was moved?

Reply:
[`no-use-before-define`](https://eslint.org/docs/rules/no-use-before-define) rule maybe?

Reply:
Yup what marc said.